### PR TITLE
Misc cleanups

### PR DESCRIPTION
**Pull request by cap10morgan**
I did most of this work a few weeks ago while trying to debug some problem I was running into. Since then, kehaar 0.5.0 was released, so I figured I'd finish upgrading to that and get this into a PR.

All the tests pass and I ran it successfully in Zie Krakenstein.

### CODE REVIEW

File: test/user_works/handlers_test.clj [hunk 1]
```diff
@@ -42,7 +44,10 @@
                                       (:id event)))))))))
 
 (deftest search-user-test
-  (let [user (create-user (async/chan)
+  (let [_ (async/take! channels/addresses
```
Comment (ericnormand) [inline test/user_works/handlers_test.clj]: why is this `take!` in the `let`?

**Comment by ericnormand** [comment]
LGTM

File: test/user_works/handlers_test.clj (hunk 1 above)
Comment (cap10morgan) [inline test/user_works/handlers_test.clj]: Yep, good call.

**Comment by cap10morgan** [comment]
@ericnormand One more quick peek before I merge?

**Comment by ericnormand** [comment]
LGTM, even better!

**Pull request merged**